pages/Surveypage.js

```diff
@@ -5,19 +5,24 @@ export class Surveypage {
   constructor(page) {
     this.page = page;
     this.createQuestionnaireButton = 'button[name="createQuestionnaire"]';
-    this.editQuestionnaire = '//button[contains(.,"Edit Questionnaire")]';
```

Comment:
The name of the file should be SurveyPage.js . Update it everywhere

Reply:
Updated the changes.